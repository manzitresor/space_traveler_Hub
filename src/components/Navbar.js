import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../assets/planet.png';

function Navbar() {
  return (
    <nav>
      <img className="nav-logo" src={logo} width={40} alt="Navbar logo" />
      <ul className="nav-items">
        <li className="nav-item-list"><NavLink to="/" activeclassname="active"><p className="tab">Rockets</p></NavLink></li>
        <li className="nav-item-list"><NavLink to="/missions" activeclassname="active"><p className="tab">Missions</p></NavLink></li>
        <li className="nav-item-list"><NavLink to="/profile" activeclassname="active"><p className="tab">Profile</p></NavLink></li>
      </ul>
    </nav>
  );
}

export default Navbar;
