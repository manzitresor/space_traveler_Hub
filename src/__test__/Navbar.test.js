import { BrowserRouter } from 'react-router-dom';
import { render, screen } from '@testing-library/react';
import Navbar from '../components/Navbar';

describe('NavBar Test', () => {
  test('If Navigations are working', () => {
    render(<BrowserRouter><Navbar /></BrowserRouter>);
    const missionText = screen.getByText('Missions');
    expect(missionText).toMatchSnapshot();
  });
});
