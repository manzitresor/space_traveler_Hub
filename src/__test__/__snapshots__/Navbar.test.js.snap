// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Test If Navigations are working 1`] = `
<p
  class="tab"
>
  Missions
</p>
`;
